version/info.go

```diff
@@ -31,6 +31,8 @@ var (
 	BuildUser string
 	BuildDate string
 	GoVersion = runtime.Version()
+	Product   = "Prometheus-common"
+	userAgent = fmt.Sprintf("%s/%s", Product, Version)
```

Comment:
Hmm, the version here is the binary version not the common version - so this could be confusing. I'd suggest putting the version at the very end.